client/src/stylesheets/base/_global.scss

```diff
@@ -1,3 +1,8 @@
 body {
+<<<<<<< HEAD
+	font-family: 'Roboto', sans-serif;
+}
+=======
```

Comment:
We'll probably want to resolve this merge conflict. I think we can keep Roboto since that's @nguyent85 said is in our wireframe.